itemloaders/__init__.py

```diff
@@ -269,25 +270,28 @@ def get_collected_values(self, field_name):
         return self._values[field_name]
 
     def get_input_processor(self, field_name):
-        """Return the input processor for the given field."""
         proc = getattr(self, '%s_in' % field_name, None)
         if not proc:
-            proc = self.get_default_input_processor_for_field(field_name)
+            proc = self._get_item_field_attr(
+                field_name,
+                'input_processor',
+                self.default_input_processor
+            )
```

Comment:
It seems this change might need an update to documentation - input and output processors can now come from item meta; this is the feature was dropped initilly (and so removed from the docs), but now it is back.

Reply:
Updated and also added a note about `dataclass`